node-disk-manager.yaml

```diff
@@ -13,33 +13,32 @@ spec:
       # that have storage attached, you could label those node and use nodeSelector.
       # Example: Label the storage nodes with - "openebs.io/nodegroup"="storage-node"
       # kubectl label node <node-name> "openebs.io/nodegroup"="storage-node"
+      hostNetwork: true
       #nodeSelector:
       #  "openebs.io/nodegroup": "storage-node"
       containers:
       - name: node-disk-manager
         command:
         - /usr/sbin/ndm
         - start
-        image: openebs/node-disk-manager:latest
-        imagePullPolicy: Always
+        image: openebs/node-disk-manager-amd64:890e662
+        imagePullPolicy: IfNotPresent
         securityContext:
           privileged: true
         volumeMounts:
-        - name: device
-          mountPath: /host/dev
-          mountPropagation: Bidirectional
         - name: mount
           mountPath: /host/mnt
-          mountPropagation: Bidirectional
+        - name: udev
+          mountPath: /run/udev
         env:
         - name: NODE_NAME
           valueFrom:
             fieldRef:
               fieldPath: spec.nodeName
       volumes:
-      - name: device
+      - name: udev
```

Comment:
can we provide some comments explaining the purpose of this volume. What properties does this volume expose. I believe this is the essence/core of this entire operator & hence is worthy of explanation.